recipes-bsp/k3-rti-wdt/k3-rti-wdt_1.1.bb

```diff
@@ -13,7 +13,7 @@ inherit dpkg
 SRC_URI = " \
     git://github.com/siemens/k3-rti-wdt.git;protocol=https;branch=master \
     file://rules"
-SRCREV = "33a6680184996074ca9731710161520edf334725"
+SRCREV = "806a7597d7853962d3eaac6340067829af0306c0"
```

Comment:
Strictly spoken, we are no longer "1.1" with this bump.

Is everything working as it should? Then I could also tag 1.2 for k3-rti-wdt.

Reply:
Yes, I tested uboot and kernel boot watchdog, watchdog reset reason can get correctly.

Reply:
Ok, v1.2 tagged and pushed.